bba/bba.go

```diff
@@ -0,0 +1,50 @@
+package bba
+
+import (
+	"sync"
+
+	"github.com/DE-labtory/cleisthenes"
+)
+
+type binary = bool
+
+const (
+	one  binary = true
+	zero        = false
+)
+
+type BBA struct {
+	*sync.RWMutex
+
+	// number of network nodes
+	n int
+	// number of byzantine nodes which can tolerate
+	f int
+
+	proposer cleisthenes.Member
+
+	// done is flag whether BBA is terminated or not
+	done bool
+	// round is value for current epoch
+	round     int
```

Comment:
Other components use "epoch" as name, how about that?

Reply:
`epoch` is for honeybadger, `round` is for BBA round. two are different. And I want to distinguish them